llm/llama-3_1/README.md

```diff
@@ -0,0 +1,300 @@
+# Serve Llama 3.1 on Your Own Infrastructure
+
+![](https://i.imgur.com/kQGzHI6.png)
+
+On July 23, 2024, Meta AI released the [Llama 3.1 model family](https://ai.meta.com/blog/meta-llama-3-1/), including a 405B parameter model in both base model and instruction-tuned forms.
+
+Llama 3.1 405B became the most capable open LLM model to date. This is **the first time an open LLM closely rivals state-of-the-art proprietary models** like GPT-4o and Claude 3.5 Sonnet.
+
+This guide walks through how to serve Llama 3.1 models **completely on your infrastructure** (cluster or cloud VPC). Supported infra:
+
+- Local GPU workstation
+- Kubernetes cluster
+- Cloud accounts ([12 clouds supported](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html))
+
+SkyPilot will be used as the unified framework to launch serving on any (or multiple) infra that you bring.
+
+## Serving Llama 3.1 on your infra
+
+We will first test the model on a GPU dev node, then package it for deployment using SkyPilot.
+
+### Step 0: Bring your infra
+
+Install SkyPilot on your local machine:
+
+```bash
+pip install 'skypilot-nightly[all]'
+```
+
+Pick one of the following depending on what infra you want to run Llama 3.1 on:
+
+**If your local machine is a GPU node**: use this command to up a lightweight kubernetes cluster:
+
+```bash
+sky local up
+```
+
+**If you have a Kubernetes GPU cluster** (e.g., on-prem, EKS / GKE / AKS / ...):
+
+```bash
+# Should show Enabled if you have ~/.kube/config set up.
+sky check kubernetes
+```
+
+**If you want to use clouds** (e.g., reserved instances): 12+ clouds are supported:
+
+```bash
+sky check
+```
+
+See [docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html) for details.
+
+### Step 1: Get a GPU dev node (pod or VM)
+
+> **Tip:** Skip directly to Step 3 to get the SkyPilot YAML to deploy Llama 3.1 on your infra.
```

Comment:
Wdym? I guess this was supposed to say "If you simply want the final deployment YAML, ..."?

Reply:
Can also link to Step 3.